### PR TITLE
hotfix -> master | Updated: performance tuning sql queries

### PR DESCRIPTION
**Pull request by we-vikram-wri231**
Test video: https://a.uguu.se/CvbltGLH.mp4

### CODE REVIEW

File: docs/guide/laravel.md [hunk 1]
```diff
@@ -204,6 +204,145 @@ Content-Type: application/json
...
+receive a result after issuing a query, and to reduce the amount of 
+resources used to process a query. 
+
+Big SQL queries can suffer performance issues. 
+Performance degradation often occurs if the database is not being properly 
+maintained or if queries can be rewritten more efficiently.
+
+Following Query optimization tips for better performance
+
+- **Apply Index On Necessary Columns:**
```
Comment (themodernpk) [inline docs/guide/laravel.md:221]: @we-vikram-wri231 use `####` for all headings

File: docs/guide/laravel.md [hunk 2]
```diff
@@ -204,6 +204,145 @@ Content-Type: application/json
...
+ We shouldn't call Eloquent relationships method in loop. 
+ First convert model object in to collection and then pass in to loop.
+ 
+  
+ **Example:**
+ ```php
+ 
+ 
+ public function category()
+     {
```
Comment (themodernpk) [inline docs/guide/laravel.md:242]: @we-vikram-wri231 fix intending https://drive.google.com/file/d/1rdxhNuiXF67HcJQBmgqcf3WK1IipEpsS/view

File: docs/guide/laravel.md [hunk 3]
```diff
@@ -204,6 +204,145 @@ Content-Type: application/json
...
+- **Use SELECT fields instead of retrieve all data:**
+
+The SELECT statement is used to retrieve data from the database. 
+In the case of large databases, it is not recommended to retrieve all data 
+because this will take more resources on querying a huge volume of data.
+
+If we execute the following query, we will retrieve all data from the Users table, 
+including, for example, users’ avatar pictures. The result table will contain 
+lots of data and will take too much memory and CPU usage.
+
```
Comment (themodernpk) [inline docs/guide/laravel.md:290]: @we-vikram-wri231 add a heading as `**BAD CODE**`

File: docs/guide/laravel.md [hunk 4]
```diff
@@ -204,6 +204,145 @@ Content-Type: application/json
...
+ ```php
+ 
+User::get();
+    
+ ```
+ 
+ Instead, you can specify the exact columns you need to get data from, thus, 
+ saving database resources. In this case, SQL Server will retrieve only the 
+ required data, and the query will have lower cost.
+
```
Comment (themodernpk) [inline docs/guide/laravel.md:300]: @we-vikram-wri231 `**GOOD CODE**` do it at other places as well

**Review by themodernpk — CHANGES_REQUESTED** [review]

File: docs/guide/laravel.md (hunk 1 above)
Comment (we-vikram-wri231) [reply docs/guide/laravel.md:221]: Ok @themodernpk Sir
I will update all changes

File: docs/guide/laravel.md (hunk 2 above)
Comment (we-vikram-wri231) [reply docs/guide/laravel.md]: @themodernpk Sir,
Fixed intending

File: docs/guide/laravel.md (hunk 3 above)
Comment (we-vikram-wri231) [reply docs/guide/laravel.md:317]: @themodernpk Sir,
Added **BAD CODE**

File: docs/guide/laravel.md (hunk 4 above)
Comment (we-vikram-wri231) [reply docs/guide/laravel.md:329]: @themodernpk Sir,
Added **GOOD CODE**

**Pull request merged**